src/displayapp/widgets/Counter.cpp

```diff
@@ -58,7 +66,32 @@ void Counter::ShowControls() {
 }
 
 void Counter::UpdateLabel() {
-  lv_label_set_text_fmt(number, "%.2i", value);
+  if (twelveHourMode) {
+    switch (value) {
+      case 0:
+        lv_label_set_text_static(number, "12");
+        break;
+      case 1 ... 12:
```

Comment:
Hey I didn't know this syntax! I can't find any doc about this in my books nor cppreference. Is this a new thing?

Reply:
It is a GCC extension, so it isn't standard C++.

Reply:
It was also used in `SettingSetTime.cpp`. It seems to be a GCC extension, not part of C++. Maybe we shouldn't use it then?

Reply:
It's maybe not that important as we support only 1 compiler anyway, but yeah... I guess I prefer to stick with the standard C++. Thans :)